### ISSUE TITLE
Error call the repository page (Public Interface)

### ISSUE THREAD

**Issue by smartexplorate** [archivesspace/archivesspace#1090]
Hello!

I cought an error if I call the repository page of a fresh installed system. Here the log

```
INFORMATION: [collection1] webapp= path=/select params={csv.escape=\&start=0&sort=repo_sort+asc&fq=-exclude_by_default:true&fq=publish:true&rows=100&q=primary_type:repository&defType=edismax&qf=four_part_id^3+title^2+finding_aid_filing_title^2+fullrecord&pf=four_part_id^4&csv.header=true&csv.encapsulator="&wt=json&facet=true} hits=0 status=0 QTime=1 
D, [2018-01-13T05:56:13.213318 #9127] DEBUG -- : Thread-3368: Responded with [200, {"Content-Type"=>"application/json", "Cache-Control"=>"private, must-revalidate, max-age=0", "Content-Length"=>"226"}, ["{\"page_size\":100,\"first_page\":1,\"last_page\":0,\"this_page\":1,\"offset_first\":1,\"offset_last\":0,\"total_hits\":0,\"results\":[],\"facets\":{\"facet_queries\":{},\"facet_fields\":{},\"facet_dates\":{},\"facet_ranges\":{},\"facet_intervals\":{}}}\n"]]... in 36ms
Jan 13, 2018 5:56:13 AM org.eclipse.jetty.server.handler.ContextHandler$Context log
INFORMATION: D, [2018-01-13T05:56:13.215151 #9127] DEBUG -- : [c5910b18-b30e-43bb-a8e2-7a17f4166483] TOTAL HITS: 0, last_page: 0

Jan 13, 2018 5:56:13 AM org.eclipse.jetty.server.handler.ContextHandler$Context log
INFORMATION: I, [2018-01-13T05:56:13.223467 #9127]  INFO -- : [c5910b18-b30e-43bb-a8e2-7a17f4166483]   Rendering repositories/index.html.erb within layouts/application

Jan 13, 2018 5:56:13 AM org.eclipse.jetty.server.handler.ContextHandler$Context log
INFORMATION: I, [2018-01-13T05:56:13.257303 #9127]  INFO -- : [c5910b18-b30e-43bb-a8e2-7a17f4166483]   Rendered shared/_pagination.html.erb (30.1ms)

Jan 13, 2018 5:56:13 AM org.eclipse.jetty.server.handler.ContextHandler$Context log
INFORMATION: I, [2018-01-13T05:56:13.261202 #9127]  INFO -- : [c5910b18-b30e-43bb-a8e2-7a17f4166483]   Rendered repositories/index.html.erb within layouts/application (35.8ms)

Jan 13, 2018 5:56:13 AM org.eclipse.jetty.server.handler.ContextHandler$Context log
INFORMATION: I, [2018-01-13T05:56:13.270102 #9127]  INFO -- : [c5910b18-b30e-43bb-a8e2-7a17f4166483] Completed 500 Internal Server Error in 157ms

Jan 13, 2018 5:56:13 AM org.eclipse.jetty.server.handler.ContextHandler$Context log
INFORMATION: F, [2018-01-13T05:56:13.283787 #9127] FATAL -- : [c5910b18-b30e-43bb-a8e2-7a17f4166483]   

Jan 13, 2018 5:56:13 AM org.eclipse.jetty.server.handler.ContextHandler$Context log
INFORMATION: F, [2018-01-13T05:56:13.285665 #9127] FATAL -- : [c5910b18-b30e-43bb-a8e2-7a17f4166483] ActionView::Template::Error (undefined method `pages' for nil:NilClass):

Jan 13, 2018 5:56:13 AM org.eclipse.jetty.server.handler.ContextHandler$Context log
INFORMATION: F, [2018-01-13T05:56:13.287706 #9127] FATAL -- : [c5910b18-b30e-43bb-a8e2-7a17f4166483]     1: <% if pager.pages.last > 1 %>
[c5910b18-b30e-43bb-a8e2-7a17f4166483]     2: <div id="paging">
[c5910b18-b30e-43bb-a8e2-7a17f4166483]     3:   
[c5910b18-b30e-43bb-a8e2-7a17f4166483]     4:   <ul class="pagination">

Jan 13, 2018 5:56:13 AM org.eclipse.jetty.server.handler.ContextHandler$Context log
INFORMATION: F, [2018-01-13T05:56:13.289254 #9127] FATAL -- : [c5910b18-b30e-43bb-a8e2-7a17f4166483
```

What is to do or what happend at the system side? 

Thanks in advance for an answer or hint!

**Comment by mark-cooper** [comment]
@smartexplorate I think you see this on a fresh system with 0 repositories. It's not elegant for sure and should be handled better, but after adding a repository it will go away.

**Comment by cfitz** [comment]
Yeah, this also confused me. I added an attempt to fix this in #1106. Also displays an error page if it cannot connect to the backend...
